### PR TITLE
fix(lang): update Japanese translations

### PR DESCRIPTION
**Pull request by Kori-Syusetu**
Fixed "unnatural Japanese sentences" and "incomprehensible translation"

## Description
Fixed "unnatural Japanese sentences" and "incomprehensible translation"

## Specific Changes proposed
lang/ja.json
Line 5,15-19,21-24

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: lang/ja.json [hunk 1]
```diff
@@ -12,15 +12,15 @@
...
   "Unmute": "ミュート解除",
-  "Playback Rate": "再生レート",
-  "Subtitles": "サブタイトル",
-  "subtitles off": "サブタイトル オフ",
-  "Captions": "キャプション",
-  "captions off": "キャプション オフ",
+  "Playback Rate": "再生速度",
+  "Subtitles": "吹き替え・字幕１",
+  "subtitles off": "吹き替え・字幕１ オフ",
+  "Captions": "表題・説明・字幕２",
```
Comment (gkatsev) [inline lang/ja.json]: We don't actually want subtitles and captions to be numbered here. Subtitles are the general term outside of the United States. Captions generally refer to Subtitles for the Hard of Hearing, that is, subtitles that also include other auditory cues.

File: lang/ja.json [hunk 2]
```diff
@@ -12,15 +12,15 @@
...
+  "captions off": "表題・説明・字幕２ オフ",
   "Chapters": "チャプター",
-  "You aborted the media playback": "動画再生を中止しました",
-  "A network error caused the media download to fail part-way.": "ネットワーク エラーにより動画のダウンロードが途中で失敗しました",
-  "The media could not be loaded, either because the server or network failed or because the format is not supported.": "サーバーまたはネットワークのエラー、またはフォーマットがサポートされていないため、動画をロードできませんでした",
-  "The media playback was aborted due to a corruption problem or because the media used features your browser did not support.": "破損の問題、またはお使いのブラウザがサポートしていない機能が動画に使用されていたため、動画の再生が中止されました",
+  "You aborted the media playback": "ユーザーにより動画再生を中断しました",
```
Comment (gkatsev) [inline lang/ja.json]: these set of changes don't seem necessary. They were created via a translation service and seem correct.

File: lang/ja.json (hunk 1 above)
Comment (Sabitakugi) [reply lang/ja.json]: Sorry for the late reply
Even the part I know
There is basically only one subtitle in Japan.
However, non-Japanese videos have multiple subtitles
In this case, multiple subtitles are language names.

File: lang/ja.json (hunk 2 above)
Comment (Sabitakugi) [reply lang/ja.json]: If there are two "or", it will basically be unnatural Japanese

**Pull request closed without merging**